### ISSUE TITLE
IE 11: startsWith not defined

### ISSUE THREAD

**Issue by h0jeZvgoxFepBQ2C** [nanostudio-org/nanogallery2#161]
one method accesses startsWith, which is not implemented in IE (all versions):

```
        // next/previous media
        if(  G.O.imageTransition.startsWith('SWIPE') ) {
          if( G.VOM.NGY2Item(-1).mediaTransition() ) {
            ViewerSetMediaVisibility(G.VOM.NGY2Item(-1), G.VOM.$mediaPrevious, 1);
          }
          if( G.VOM.NGY2Item(1).mediaTransition() ) {
            ViewerSetMediaVisibility(G.VOM.NGY2Item(1), G.VOM.$mediaNext, 1);
          }
```

For IE a polyfill has to be added to make it work:

```
if (!String.prototype.startsWith) {
  String.prototype.startsWith = function(searchString, position) {
    position = position || 0;
    return this.indexOf(searchString, position) === position;
  };
}
```

**Comment by Kris-B** [comment]
will be integrated in the next release, thanks !

**Comment by Kris-B** [comment]
fixed in v2.4.0 - thx!

**Comment by Kris-B** [comment]
Please note that package for v2.4.0 was corrupted - please use v2.4.1
